### DISCUSSION TITLE
Download all missing custom nodes

### DISCUSSION THREAD

**Discussion by AI-taja** [Comfy-Org/ComfyUI-Manager#752] [Ideas]
A lot of the times when using the manager for clean installs of Comfy, I need to install each custom node separately.
A button that just installs all of them at once would be really useful.

**Comment by ltdrdata** [comment]
1. goto missing node dialog 
2. check one of them
3. select all checkbox
4. click install button

It will install all selected nodes.

FYI, a UX upgrade PR is currently in progress, so this behavior will change.
https://github.com/ltdrdata/ComfyUI-Manager/pull/744
